packages/core/src/controllers/SwapController.ts

```diff
@@ -202,7 +202,7 @@ export const SwapController = {
       invalidSourceTokenAmount,
       availableToSwap:
         caipAddress && !invalidToToken && !invalidSourceToken && !invalidSourceTokenAmount,
-      isAuthConnector: type === 'AUTH'
+      isAuthConnector: type === 'ID_AUTH'
```

Comment:
Should we maybe add `ID_AUTH` to a constants file ?

Reply:
this should be type checked already else it'd say that it would always be false 🤔 
but we should use the constant regardless IMO